annotation/compiler/src/main/java/com/bumptech/glide/annotation/compiler/ProcessorUtil.java

```diff
@@ -48,7 +48,6 @@
 import javax.lang.model.type.TypeMirror;
 import javax.lang.model.type.TypeVariable;
 import javax.lang.model.util.ElementFilter;
-import javax.lang.model.util.Elements;
```

Comment:
The lines removed in this file are still needed for projects that use the support library, including internal projects.